src/pages/index.js

```diff
@@ -274,17 +246,43 @@ export const pageQuery = graphql`
         contentPadding
       }
     }
+    allMarkdownRemark(
+      sort: {
+        fields: [frontmatter___releaseDate, frontmatter___title]
+        order: [DESC, ASC]
+      }
+      filter: { fields: { slug: { regex: "/whats-new/" } } }
```

Comment:
I mentioned it above in this file: if we only need three posts, let's limit the query. That should speed this up a little bit.

```suggestion
      sort: {
        fields: [frontmatter___releaseDate, frontmatter___title]
        order: [DESC, ASC]
      }
      filter: { fields: { slug: { regex: "/whats-new/" } } }
      limit: 3
```